<ui version="4.0" >
 <class>DownloadItem</class>
 <widget class="QWidget" name="DownloadItem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="fileIcon" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Ico</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QLabel" name="label_2" />
     </item>
     <item>
      <widget class="SqueezeLabel" native="1" name="fileNameLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" stdset="0" >
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="downloadInfoLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="tryAgainButton" >
     <property name="toolTip" >
      <string>Try to resume download</string>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="stopButton" >
     <property name="toolTip" >
      <string>Stop downloading</string>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="openButton" >
     <property name="toolTip" >
      <string>Open item</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>://bullet_triangle_green.png</normaloff>://bullet_triangle_green.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteButton" >
     <property name="toolTip" >
      <string>Delete item</string>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>://delete.png</normaloff>://delete.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="horizontalLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetMinimumSize</enum>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqueezeLabel</class>
   <extends>QWidget</extends>
   <header>squeezelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="data/data.qrc" />
 </resources>
 <connections/>
</ui>
